src/plugins/postcss-icss-parser.js

```diff
@@ -4,62 +4,67 @@ import { urlToRequest } from 'loader-utils';
 
 const pluginName = 'postcss-icss-parser';
 
+function normalizeIcssImports(icssImports) {
+  return Object.keys(icssImports).reduce((accumulator, url) => {
+    const tokensMap = icssImports[url];
+    const tokens = Object.keys(tokensMap);
+
+    if (tokens.length === 0) {
+      return accumulator;
+    }
+
+    const normalizedUrl = urlToRequest(url);
+
+    if (!accumulator[normalizedUrl]) {
+      // eslint-disable-next-line no-param-reassign
+      accumulator[normalizedUrl] = tokensMap;
+    } else {
+      // eslint-disable-next-line no-param-reassign
+      accumulator[normalizedUrl] = {
+        ...accumulator[normalizedUrl],
```

Comment:
@evilebottnawi this is a relatively new feature in javascript.  Introducing this broke our webpack build.  Any chance you could use Object.assign(...) instead?

Reply:
https://github.com/webpack-contrib/css-loader/pull/1017#issuecomment-565471368